### ISSUE TITLE
QEMU Hard Disk Image on Host System

### ISSUE THREAD

**Issue by singhamitoj** [foxlet/macOS-Simple-KVM#334]
I am curious about the qemu disk image. Should you be able to flash the hard disk image to an external drive, and boot to it from my host (Basically, another way to hackintosh)? The image uses qcow2, so I am not sure if it will work.

**Comment by homandiy** [comment]
I don't think that is a problem. I am using WSL2. My other drive is /mnt/d/. I create image dMac.qcow2 at that drive. And I modify the basic.sh to /mnt/d/dMac.qcow2.

**Comment by RePrete** [comment]
If I understood correctly your question, the answer should be no.
After installing macos on an external drive (passed changing the config for the devices) you shouldn't be able to boot directly from the disk.
This is because during the installation (and the normal usage) qemu is abstracting (emulating) the whole machine where macos is running, and this fix all the compatibility issues with the host machine.

Booting directly from the hdd will result in no abstraction of the host machine and will lead 99% in some compatibility issues.
Also probably the boot/secureBoot will give you some errors.
Moreover probably some system files/drivers could be missing because not needed to the virtual machine during the installation.


You could try, but probably is easier to do a clean hackintosh installation.

**Comment by singhamitoj** [comment]
I flashed the macOS base system onto a usb drive and I booted to macOS. It worked!

**Comment by RePrete** [comment]
So you did the installation using vm, loaded the specific config for clover/opencore in the EFI partition and booted that partition directly from BIOS after?

**Comment by singhamitoj** [comment]
No, I did not do that. I went into the files for the vm, and got to the BaseSystem.img. I flashed it onto a USB stick to create a bootable installer, and I restarted the computer and booted into macOS.
